bindata/network/ovn-kubernetes/004-config.yaml

```diff
@@ -9,3 +9,27 @@ data:
   net_cidr:   {{.OVN_cidr}}
   svc_cidr:   {{.OVN_service_cidr}}
   k8s_apiserver: "{{.K8S_APISERVER}}"
+
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: env-overrides
+  namespace: openshift-ovn-kubernetes
+  annotations:
+    # This is for debugging purposes and will be updated external to the operator
+    networkoperator.openshift.io/create-only: ""
```

Comment:
oh, so hey, it turns out that it's accidentally undocumented, but you can actually do

    volumes:
    - name: env-overrides
      configMap:
        name: env-overrides
        optional: true

and then it won't complain if the configmap doesn't exist. So we don't actually need to create the default empty `env-overrides` configmap...

Reply:
Cool, I'll move this to the README.